README.md

```diff
@@ -104,7 +104,7 @@ At that point, someone will work with you on doing a code review (typically pret
 
 ### Stay Scoped
 
-Try to keep your PRs limited to one particular issue and don't make changes that are out of scope for that issue. If you notice something that needs attention but is out-of-scope, put a TODO, FIXME, or NOTE comment above it
+Try to keep your PRs limited to one particular issue and don't make changes that are out of scope for that issue. If you notice something that needs attention but is out-of-scope, [please create a new issue.](https://github.com/rubyforgood/diaper/issues/new)
```

Comment:
@armahillo @seanmarcia Are you ok with moving to Github Issues to track required work?

Reply:
I'm totally fine with it!

Sorry things with the project are moving a little slow right now -- there is generally a slump in activity after our event before things start picking up. Also, Elisheba just got married (woo hoo) and I know Aaron is doing a bunch of traveling with his kids (he's actually going to be staying with me in about a week and then a week after that!)

We just put the call out for people interested in fellowships so hopefully we'll have dedicated people to all the projects soon :)

Reply:
Yay! What exciting news! And no worries on the speed - this is an issue I've been sitting on without working for almost a month :grimacing: